src/js/footer/090-chameleon.js

```diff
@@ -0,0 +1,11 @@
+//scroll to anchors
+document.querySelectorAll('a[href^="#"]').forEach( function( anchor ){
```

Comment:
We'll have to refactor that to `document...` something for IE8

Reply:
yup most of my JS going to be thrown out...just going to use what you have done